docs/templates/content/90-releasenotes.ftl

```diff
@@ -6,6 +6,7 @@
 ### Changed features
 * Users are now allowed to [upload source](../properties/configuration/signature scanner/#upload-source-mode) when only [license search](../properties/configuration/signature scanner/#signature-scanner-license-search) is provided. (IDETECT-1894)
 * (IDETECT-1853) Detect is now compatible with Yocto 3.0.
+* (IDETECT-1876) Detect will stop if docker is enabled on Windows.
```

Comment:
Will stop? Will it fail?

Reply:
I used stop because it is a hard stop unlike a normal detect failure which continues. For example if a detector fails, detect does not stop. Docker will force an exception and stop the run.